airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -157,7 +156,7 @@
 ]
 
 SCHEDULES = [
-    ScheduleDefinition(job=add_new_metadata_partitions, cron_schedule="* * * * *"),
+    ScheduleDefinition(job=add_new_metadata_partitions, cron_schedule="*/5 * * * *", tags={"dagster/priority": HIGH_QUEUE_PRIORITY}),
```

Comment:
Pretty much.

Because jobs were taking so long for spin up time during the registry entry back fill our queue became huge

Then it emptied slowly because these 1 minute sensors were pilling up to be ran.

This is a change that should reduce the number of sensors that can clog the queue.

Still not a fix to the resource problem though.

Dagster is recommending we move to hybrid to get more performance.